futures-util/src/io/mod.rs

```diff
@@ -26,6 +18,9 @@ pub use self::copy_into::CopyInto;
 mod flush;
 pub use self::flush::Flush;
 
+// mod lines;
```

Comment:
Mind adding a `TODO` or something here to indicate that this is wanted, but not yet implemented?